scripts/publish-rhcc-image.sh

```diff
@@ -59,15 +59,9 @@ BASE_VERSION=${VERSION%-*}
 OUTPUT_IMAGE=scan.connect.redhat.com/${PROJECT_ID}/unused:${BASE_VERSION}-${BUILD}
 fi
 
-# Ensure image is available locally to be pushed
-docker pull ${INTERNAL_IMAGE_NAME}
-
-docker tag ${INTERNAL_IMAGE_NAME} ${OUTPUT_IMAGE}
+# Copy image from internal to external repo
 echo @@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
-echo Pushing ${INTERNAL_IMAGE_NAME}
-echo as ${OUTPUT_IMAGE}
+echo Copying ${INTERNAL_IMAGE_NAME}
+echo to ${OUTPUT_IMAGE}
 echo @@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
-
-docker push ${OUTPUT_IMAGE}
-docker rmi ${INTERNAL_IMAGE_NAME}
-docker rmi ${OUTPUT_IMAGE}
+skopeo copy ${INTERNAL_IMAGE_NAME} ${OUTPUT_IMAGE}
```

Comment:
Similar issue to the above - the publish-rhcc docker job always runs on the "openshift" agent, in this case because it indirectly depends on several metadata and credential files which only exist on that agent. Do you think there's any chance skopeo will build on a RHEL7 box? Otherwise we need to rethink this script a bit (which it certainly is overdue for).

Reply:
I've just gone ahead and removed this file after your recent work on CBD-5177/rhcc-certify-and-publish.sh